tests/resolver/test_testset_resolver.py

```diff
@@ -178,6 +178,7 @@ def test_resolve_collection_data_rarity_sniffer(self):
 
         assert len(rows) == 10_000
 
+    @pytest.mark.skip(reason="Needs OpenSea API key to be set up")
```

Comment:
@vickygos I don't have permissions to set up `OS_API_KEY` env var in the project so that this test would pass again - can you set it up?
Alternatively we could just skip this test in CI.

Reply:
discussed offline and we will only skip this in CI